botocore/crt/auth.py

```diff
@@ -54,9 +54,7 @@ def add_auth(self, request):
         if self.credentials is None:
             raise NoCredentialsError()
 
-        # Use utcnow() because that's what gets mocked by tests, but set
-        # timezone because CRT assumes naive datetime is local time.
-        datetime_now = datetime.datetime.utcnow().replace(
+        datetime_now = datetime.datetime.now(datetime.timezone.utc).replace(
             tzinfo=datetime.timezone.utc
         )
```

Comment:
No need to call `.replace()` here anymore; it's already set.

```suggestion
        datetime_now = datetime.datetime.now(datetime.timezone.utc)
```